src/main/java/seedu/address/model/participant/Participant.java

```diff
@@ -22,9 +24,10 @@
  */
 public class Participant extends Person {
```

Comment:
Is there still a need for the Person class?

Reply:
for this application like what we discussed we want to have a list of Participant objects since it contains more information than Person, it is arguable if we should change the class Person to Participant entirely but I think it is still useful for us to keep it for possible future implementation.

Also NextOfKin is its own separate class which also support your argument further so I think until the last submission the sole purpose of Person is for Participant to inherit it as a legacy code